effects/butterfly-gears/data/textureloop-generated.txt

```diff
@@ -0,0 +1,74 @@
+;>>> startrow
+	move.l	d5,d0		; start new row
+;>>> plot-even
+	move.l  d0,d3           ; VVVVVVVvvvvvvvvv UUUUUUUuuuuuuuuu
+	lsr.w   d2,d3           ; VVVVVVVvvvvvvvvv ---------UUUUUUU
```

Comment:
For the reference:
 * [A brief discussion about texturemapper innerloops using ADDX](https://amycoders.org/opt/innerloops.html)
 * [MC68000 Instructions timing](http://linas.org/mirrors/www.nvg.ntnu.no/2002.09.16/amiga/MC680x0_Sections/mc68000timing.HTML)
I guess you already know these sites...

Reply:
Good point! In the original effect I wanted to avoid negative offsets in the texture-relative data register (hence 7-bit U/Vs instead of 8) and never bothered to optimize it further…

With an `lsr`-free inner loop, there’s now room for one more rotzoom object, yay!